pod/perlfunc.pod

```diff
@@ -4249,7 +4249,7 @@ This makes it easy to get a month name from a list:
 
     my @abbr = qw(Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec);
     print "$abbr[$mon] $mday";
-    # $mon=9, $mday=18 gives "Oct 18"
+    # $mon=9, $mday=18 gives "Oct 18" (not "Sep 18"!)
```

Comment:
That's not even about `localtime`, that's just basic array indexing. Why do we need to add a special note for that?

Reply:
Ｉ quickly checked for the name of the month field.

And ended up giving all 26 pages with the wrong date and now it's too late to change it.

Reply:
The "name" of the month field is "index 4 of the list returned by localtime", which is not shown in this code snippet.

The description of `$mon` (just above the code snippet you patched) is 

> $mon the month in the range 0..11, with 0 indicating January and 11 indicating December.

Besides, if you're generating human-readable timestamps, you should be using `POSIX::strftime`, which handles all the details.

(And if you're submitting documents to a court and doing date math/formatting the manual and tedious way, I hope you actually read the description of each field and don't just skim random examples.)

That said, I do think the description should encourage use of `strftime` more and explicitly mention the difference between 1-based `$mday` and 0-based `$mon`. Hence #22749.